packages/legacy/app/package.json

```diff
@@ -68,6 +68,7 @@
     "react-native-gifted-chat": "*",
     "react-native-keychain": "^8.1.1",
     "react-native-localize": "^2.2.4",
+    "react-native-logs": "^5.1.0",
```

Comment:
I noticed the dep here is `5.1.0` and the dep in `@hyperledger/aries-bifold-remote-logs` is `5.0.1`, probably won't affect anything but might be good to get them in sync in the future in case they ever interact

Reply:
Good catch. I took the opportunity and updated to react-native-logs 5.1.0 across the board.